app/services/browser_support.rb

```diff
@@ -0,0 +1,62 @@
+class BrowserSupport
+  @cache = LruRedux::Cache.new(1_000)
+
+  BROWSERSLIST_TO_BROWSER_MAP = {
+    and_chr: :chrome?,
+    and_uc: :uc_browser?,
+    chrome: :chrome?,
+    edge: :edge?,
+    firefox: :firefox?,
+    ios_saf: :safari?,
+    op_mini: :opera_mini?,
+    opera: :opera?,
+    safari: :safari?,
+    samsung: :samsung_browser?,
+  }.transform_values { |method| Browser::Base.instance_method(method) }.
```

Comment:
support for procs too
```suggestion
  }.transform_values(&:to_proc).
```

Reply:
> support for procs too

Added as part of 86c4df0c1.